### PR TITLE
Create GC and TC members and charter

### PR DESCRIPTION
**Pull request by DavidPHirsch**
Signed-off-by: David Hirsch <david.hirsch@dynatrace.com>

### CODE REVIEW

**Review by AloisReitbauer — CHANGES_REQUESTED** [review]
we should just provide the names for GC members and probably the GH handle

**Comment by agardnerIT** [comment]
The technical committee sounds potentially interesting. What is the charter / AOR?

@bradmccoydev and I were talking about this the other day. Interestingly Brad has already had a lot of support for him to join the TC: https://github.com/keptn/community/issues/238

File: governance/members-and-charter.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+ Role and responsibilities
+
+
+ ### TC Members
+ 
+ @thisthat
+ 
+ @bradmccoydev 
+
```
Comment (thschue) [inline governance/members-and-charter.md:76]:
```suggestion
@thschue 
```

**Comment by DavidPHirsch** [comment]
@AlexsJones @AloisReitbauer @AnaMMedina21 could you please review this issue?

File: governance/members-and-charter.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+ The Governance Committee has a set of rights and responsibilities including the following:
+
+ Charter for Technical Committee.
+ Define, evolve, and defend a Code of Conduct, which must include a neutral, unbiased process for resolving conflicts.
+ Define and evolve project governance structures and policies, including how contributors become committers/maintainers, approvers, reviewers, members, etc.
+ Manage and organize the election process.
+ Logo/landing page/marketing.
+ Making a final decision if a consensus cannot be reached (see _Decision Making_)
+ Maintain relationships with the CNCF. For instance, creating documents describing the project
```
Comment (AlexsJones) [inline governance/members-and-charter.md:23]: Missing a full stop at the end. 

Generally it's quite a busy paragraph, might want to break it down a bit for readability

File: governance/members-and-charter.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+ The following Governance Committee's term goes from the December 1st 2022 until November 30th 2023.
+ You can find the election process detailed [here.](https://github.com/keptn/community/blob/main/elections/2022/governance-committee-election.md)
+
+ ### Alex Jones
+
+ - Company: Canonical
+ - GitHub: https://github.com/AlexsJones
+ - Twitter: https://twitter.com/AlexJonesax
+ - Personal Website: https://www.cloudnativeskunkworks.io/
```
Comment (AlexsJones) [inline governance/members-and-charter.md:43]: Please remove website, it's no longer used

**Review by AnaMMedina21 — APPROVED** [review]

**Pull request merged**